workers/ohsome_quality_analyst/base/indicator.py

```diff
@@ -166,14 +166,15 @@ def _get_default_figure(self) -> str:
         return svg_string.getvalue()
 
     def create_html(self):
-        template_folder = r".\templates"
-        template_filename = "indicator_schema.html"
-        script_path = os.path.dirname(os.path.abspath(__file__))
-        template_path = os.path.join(script_path, template_folder)
+        template_path = os.path.join(
+            os.path.dirname(os.path.abspath(__file__)),
+            "templates",
+            "indicator_schema.html",
+        )
         env = jinja2.Environment(
             loader=jinja2.FileSystemLoader(template_path),
         )
```

Comment:
```suggestion
        template_dir = os.path.join(
            os.path.dirname(os.path.abspath(__file__)),
            "templates",
        )
        env = jinja2.Environment(
            loader=jinja2.FileSystemLoader(template_dir),
        )
```
FileSystemLoader expects a path to a directory.